test/specs/suite2.spec.js

```diff
@@ -0,0 +1,168 @@
+    let randomIndex2 = Math.floor(Math.random() * 5);
+    let randomIndex3 = Math.floor(Math.random() * 4);
+
+    let arrayOfAdded = [];
+    let obj1 = {
+      title: await inventoryPage.arrayOfTitles[randomIndex1],
+      price: await inventoryPage.arrayOfPrices[randomIndex1],
+      descr: await inventoryPage.arrayOfDescr[randomIndex1],
+    };
+    let obj2 = {
+      title: await inventoryPage.arrayOfTitles[randomIndex2],
+      price: await inventoryPage.arrayOfPrices[randomIndex2],
+      descr: await inventoryPage.arrayOfDescr[randomIndex2],
+    };
+    let obj3 = {
+      title: await inventoryPage.arrayOfTitles[randomIndex3],
+      price: await inventoryPage.arrayOfPrices[randomIndex3],
+      descr: await inventoryPage.arrayOfDescr[randomIndex3],
+    };
+
+    arrayOfAdded.push(obj1);
+
+    await inventoryPage.selectItem(randomIndex1);
+
+    if (randomIndex1 != randomIndex2) {
+      arrayOfAdded.push(obj2);
+      await inventoryPage.selectItem(randomIndex2);
+    } else if (randomIndex1 != randomIndex3) {
+      arrayOfAdded.push(obj3);
+      await inventoryPage.selectItem(randomIndex3);
+    }
+
+    let invArrayLength = arrayOfAdded.length;
+
+    await inventoryPage.cartBtn.click();
+    
+    let arrayOfAddedInCart = [];
+
+    for (let i = 0; i < invArrayLength; i++) {
+      let objCart = {
+        title: await shoppingCart.arrayOfTitles[i],
+
+        price: await shoppingCart.arrayOfPrices[i],
+
+        descr: await shoppingCart.arrayOfDescr[i],
+      };
+
+      arrayOfAddedInCart.push(objCart);
+    }
+
+    await expect(arrayOfAdded).toMatchObject(arrayOfAddedInCart);
+
+    for (let i = 0; i < invArrayLength; i++) {
+      await shoppingCart.removeBtn.click();
+    }
+  });
+  it("should verify calculated price", async () => {
+    let randomIndex1 = Math.floor(Math.random() * 6);
+    let randomIndex2 = Math.floor(Math.random() * 5);
+    let randomIndex3 = Math.floor(Math.random() * 4);
+    let arrayOfAdded = [];
+    let obj1 = {
+      title: await inventoryPage.arrayOfTitles[randomIndex1],
+      price: await inventoryPage.arrayOfPrices[randomIndex1],
+      descr: await inventoryPage.arrayOfDescr[randomIndex1],
+    };
+    let obj2 = {
+      title: await inventoryPage.arrayOfTitles[randomIndex2],
+      price: await inventoryPage.arrayOfPrices[randomIndex2],
+      descr: await inventoryPage.arrayOfDescr[randomIndex2],
+    };
+    let obj3 = {
+      title: await inventoryPage.arrayOfTitles[randomIndex3],
+      price: await inventoryPage.arrayOfPrices[randomIndex3],
+      descr: await inventoryPage.arrayOfDescr[randomIndex3],
+    };
+
+    arrayOfAdded.push(obj1);
+
+    await inventoryPage.selectItem(randomIndex1);
+    
+    if (randomIndex1 != randomIndex2) {
+      arrayOfAdded.push(obj2);
+      await inventoryPage.selectItem(randomIndex2);
+      
+    } else if (randomIndex1 != randomIndex3) {
+      arrayOfAdded.push(obj3);
+      await inventoryPage.selectItem(randomIndex3);
+    }
```

Comment:
we have the same logic above, so we can move it separate module like randomiser or smth to avoid code duplication